docs/server-side/v2/api-reference/stream-key/disable-rtmp-stream-key.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Disable RTMP stream key
+nav: 3.74
+---
+
+Use this request to disable the RTMP stream key for a specific room.
+
+<PostRequest title="https://api.100ms.live/v2/stream-keys/room/<room_id>/disable" />
+<Request id="req-0">
+
+```bash
+curl --location --request POST 'https://api.100ms.live/v2/stream-keys/room/<room_id>/disable' \
+--header 'Authorization: Bearer <management_token>'
+```
+</Request>
+
+<ResponseBox id="res-0" status="201 OK">
+```json
+{
+    "message": "key disabled successfully"
+}
+```
+</ResponseBox>
+
+### Arguments
+
+| Field   | Type     | Description |
+|:--------|:---------|:------------|
+| room_id | `string` | mandatory   |
+
+### Errors
+
+| Description                   | Code |
+|:------------------------------|:-----|
+| Key not found                 | 404  |
+| Room not found                | 404  |
+| Room is disabled (not active) | 403  |
```

Comment:
Disabled Room is not just non-active rooms right? basically they are locked and cant be moved to active state until enabled (unlocked) - have we mentioned it as non-active rooms in other places also? else we can describe them as locked rooms, or just disabled

Reply:
> Disabled Room is not just non-active rooms right? basically they are locked and cant be moved to active state until enabled (unlocked)

Correct, these are disabled rooms which need to be enabled.
Also, when you say non-active rooms, you mean the session is not running in them, right? We can start RTMP In for those rooms. I tested this.